src/main/java/org/gridsuite/shortcircuit/server/entities/ShortCircuitParametersEntity.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.entities;
+
+import com.powsybl.shortcircuit.InitialVoltageProfileMode;
+import com.powsybl.shortcircuit.StudyType;
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.Accessors;
+import org.gridsuite.shortcircuit.server.dto.ShortCircuitPredefinedConfiguration;
+
+import java.util.UUID;
+
+/**
+ * @since 1.7.0
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+@Getter
+@Setter
+@Entity
+@Table(name = "shortcircuit_parameters")
```

Comment:
```suggestion
@Table(name = "short_circuit_parameters")
```
in order for the data migration script to work, the parameters table must have the same name as the one defined in study-server

Reply:
Problem fixed in gridsuite/deployment#417